python/lsst/obs/base/_instrument.py

```diff
@@ -600,12 +600,18 @@ def makeExposureRecordFromObsInfo(
     if (k := "azimuth") in supported:
         extras[k] = azimuth
 
+    if "group" in dimension.implied:
+        extras["group"] = obsInfo.exposure_group
+    elif "group_name" in supported:
+        extras["group_name"] = obsInfo.exposure_group
+        extras["group_id"] = obsInfo.visit_id
+    else:
+        raise RuntimeError(f"Unable to determine where to put group metadata in exposure record: {supported}")
```

Comment:
Would this also make it an error to have a dimension universe with no `group` at all?

Reply:
Yes, but I'm not sure that's a problem given we are mostly assuming our exposure record her anyhow and anybody that's not using our exposure record will be overriding this in an ingest task subclass anyhow.